OJP/OJP_Trips.xsd

```diff
@@ -878,6 +895,172 @@
 			<xs:element name="TripFare" type="TripFareResultStructure" minOccurs="0" maxOccurs="unbounded"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="UserPreferencesStructure">
+		<xs:annotation>
+			<xs:documentation>A collection of user preferences to allow the algorithm to narrow-down multimodal trip combinations to those that may be suitable for the user.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="LowPrice" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference that the trip has a low price.</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="important"/>
+						<xs:enumeration value="average"/>
+						<xs:enumeration value="unimportant"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="Speed" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference about the speed/short duration of the trip.</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="important"/>
+						<xs:enumeration value="average"/>
+						<xs:enumeration value="unimportant"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="Comfort" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference about the comfort/convenience of the trip, including few changes, short transfers and convenient vehicle types. Reflect USER PREFERENCE from Transmodel</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="premium"/>
+						<xs:enumeration value="average"/>
+						<xs:enumeration value="basic"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="Sportiness" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's endurance and fitness for sportive, self-powered trip legs (walk, cycle, etc.).</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="high"/>
+						<xs:enumeration value="average"/>
+						<xs:enumeration value="low"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="EnvironmentalSafety" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
```

Comment:
_[This extra text could be added to give more hints about the meaning and how it should be implemented]_
If high, favour "green" modes/lines such as bike sharing and (electric) trains, avoid or restrict modes/lines
known for higher CO2 emissions such as (conventional) taxi, ridehailing or coach.
If low, basically ignore (same as average).
(Probably better to turn it into a true/false option)